mlflow/tracing/constant.py

```diff
@@ -34,3 +34,16 @@ class SpanAttributeKey:
 # Key for the trace schema version in the trace. This key is also used in
 # Databricks model serving to be careful when modifying it.
 TRACE_SCHEMA_VERSION_KEY = "mlflow.trace_schema.version"
+
+# The list of tags generated from resolve_tags that are required for tracing UI
+TRACE_RESOLVE_TAGS_ALLOWLIST = (
```

Comment:
Can we use the constants defined in `mlflow/utils/mlflow_tags.py`?

Reply:
good suggestion. have incorporated the change